deploy/kubernetes/elastic-agent-standalone-kubernetes.yaml

```diff
@@ -305,17 +303,40 @@ data:
               pattern: ^\s
               match: after
             processors:
-              - add_fields:
-                  target: ''
-                  fields:
-                    ecs.version: 1.12.0
+              - add_locale: null
+            ignore_older: 72h
+      - id: windows-event-log
```

Comment:
Any background on this change?

Reply:
@elastic/elastic-agent-data-plane can you give us some more info about this change? 

We updated our standalone manifests to include your last package version and we are in the progress of aligning the manifest that is auto-generated from kibana UI with what we have in github. So we removed the `add_field` processor  but we added the `add_locale` that we found. Can you give us some more info? Is it needed?

Reply:
The `condition: '${host.platform} == ''windows''' will evaluate to false for every stream and prevent this from actually starting.

For Kubernetes you can omit this since Windows isn't supported. The policy generated by Fleet is universal for all operating systems.

Reply:
You want to align this as closely as you can with the latest version of the system package generated by Fleet, as that is what is actually maintained.

I think there is a problem here where this is naturally going to drift out of sync with the latest version of the system package unless we either automate updates to it or do the manual work to sync it.

What do you actually want to have in these manifests? What is the goal? It seems like there would be less chance of regression if we only included a minimal configuration that is kept up to date. 

Is it worth including only the Kubernetes integration here and omitting the system integration entirely?

Reply:
@cmacknz Sorry for not replying you before, Indeed on one hand we have the problem of syncing and on the other one we want to have it minimal especially no automation is implemented at the moment.

For the moment we agreed to keep the best effort and manually do the sync and also to keep system integration for now.

| Is it worth including only the Kubernetes integration here and omitting the system integration entirely?

This is something that might reconsider